test/lib/ffi/ffi_convert.lua

```diff
@@ -769,6 +770,18 @@ do
   x.ppf = ffi.C.strcpy
 end
 
+do
```

Comment:
the indentation is really messy (even a tab is there)
otherwise i couldnt really figure out whats the purpose of it, but that can be my fault

Reply:
Thanks for the review.  Fixed with #49 and #50.

Reply:
thx for the fix! :D (i try to catch up, so actually i do it whenever i have time for it, but there is still much to discover and understand to become a more valuable reviewer :D )